index2.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="projects.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+  <title>works section</title>
+</head>
+<body>
+  <section class="works"> 
```

Comment:
- [ ] This section should have been styled with CSS grid. Please add the `display: grid` and style your section based on this since this is a mandatory project requirement.
 - [ ] This section has a different background color in the Figma template that makes the project cards pop and look like they are floating above the section. Please add that background color the same as the Figma template.